app/views/documents/_recommendation_list.html.erb

```diff
@@ -1,9 +1,9 @@
-<p class="text-wrap text-left text-sm mb-2r">This recommendation information was generated by a Large
+<p class="text-wrap text-left text-sm mb-2r">This suggestion information was generated by a Large
```

Comment:
Removing information ("This suggestion was ...") is more concise. Not super sure where this is appearing, so defer to you.